scripts/qa_crowbarsetup.sh

```diff
@@ -1831,16 +1838,25 @@ function custom_configuration()
             proposal_set_value cinder default "['attributes']['cinder']['volumes'][0]['backend_driver']" "'${cinder_backend}'"
             case "$cinder_backend" in
                 netapp)
-                    cinder_netapp_proposal_configuration
+                    cinder_netapp_proposal_configuration "0"
                     ;;
             esac
 
             # add a second backend to enable multi-backend, if not already present
             if [[ $want_cindermultibackend = 1 ]] ; then
-                if ! crowbar cinder proposal show default | grep -q local-multi; then
+                # in case of testing netapp, add a second backend with a different storage protocol
+                if [[ $cinder_backend = "netapp" ]]; then
+                    if [[ $cinder_netapp_storage_protocol = "iscsi" ]] ; then
+                        cinder_netapp_storage_protocol="nfs"
+                    else
+                        cinder_netapp_storage_protocol="iscsi"
```

Comment:
might be better to have this value as explicit param to the cinder_netapp_proposal_configuration function